### ISSUE TITLE
Linux binary in the repo?

### ISSUE THREAD

**Issue by avsej** [uber-common/cpustat#20]
What the reason of having compiled binary in the repo? `getfamily/getdelays`. It is compiled for one particular distribution and architecture. Are you going to add others?

```
$ file getfamily/getdelays 
getfamily/getdelays: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked (uses shared libs), for GNU/Linux 2.6.24, BuildID[sha1]=74b46a4149235081aa3cc4fa9156a8e09b9ad78e, not stripped
$ ldd getfamily/getdelays 
	linux-vdso.so.1 =>  (0x00007ffc07ebe000)
	libc.so.6 => /lib64/libc.so.6 (0x00007f7d6ecde000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f7d6f0ab000)
```

**Comment by mranney** [comment]
Whoops. That was a mistake. I'll pull it out. Thanks for pointing this out.

**Comment by mranney** [comment]
Fixed by c30cba6f586bf9f36450d40b8e8fa199c460d00f
